cypress/e2e/models/migration/archetypes/archetype.ts

```diff
@@ -325,4 +325,22 @@ export class Archetype {
         clickKebabMenuOptionArchetype(this.name, "Discard review");
         confirm();
     }
+    clickAssessButton() {
+        Archetype.open();
+        clickKebabMenuOptionArchetype(this.name, "Assess");
+    }
+    discardAssessments(): void {
+        this.clickAssessButton();
+        Assessment.discardAssessments();
+    }
+    verifyAssessmentTakeButtonEnabled(): void {
+        //validates current page
+        cy.get("section.pf-v5-c-page__main-section.pf-m-light h1").then((h1) => {
+            if (!h1.text().includes("Assessment Actions")) {
+                Archetype.open();
+                this.clickAssessButton();
```

Comment:
You are using same code in several places. Consider creating function for it, taking into account that it can be used in more cases with respective parameters

Reply:
done